frontend/src/metabase/nav/components/WhatsNewNotification/utils.unit.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { createMockVersionInfoRecord } from "metabase-types/api/mocks";
+import { getLatestEligibleReleaseNotes } from "./utils";
+
+describe("What's new - utils", () => {
+  describe("getLatestEligibleReleaseNotes", () => {
```

Comment:
Please add tests according to the testing plan

Reply:
Actually everything is/should already be tested in the component, I wrote these tests to help me write the util function without having to think about the component.
Should I remove these tests to avoid the duplication and confusion?

Reply:
There are no tests for 
1 - for embedding nothing is returned even if all conditions are otherwise correct
2 - if the current version is not defined
3 - if there is no version record for the current version available